packages/template-retail-react-app/app/hooks/use-auth-modal.js

```diff
@@ -97,13 +97,17 @@ export const AuthModal = ({
 
         const handlePasswordlessLogin = async (email) => {
             try {
-                await authorizePasswordlessLogin.mutateAsync({userid: email})
+                const res = await authorizePasswordlessLogin.mutateAsync({userid: email})
+                if (res.status !== 200) {
+                    const errorData = await res.json()
+                    throw new Error(`${res.status} ${errorData.message}`)
+                }
```

Comment:
Have we considered moving the logic to throw the error message into commerce-sdk-react?

Reply:
We could... Is there a specific reason to move this to the sdk?

Reply:
I guess it would be to simplify the code within the retail react app, if we give the responsibility of throwing the error to the sdk the retail react app doesn't have to worry about checking the status code and can just catch the error when it occurs.

it'll also reduce code duplication, for example for passwordless checkout we wouldn't need to duplicate this logic

Reply:
+1 on moving it to the sdk